addons/centered/src/html.js

```diff
@@ -1,34 +1,42 @@
 import { document, Node } from 'global';
+import styles from './styles';
 
-const style = {
-  position: 'fixed',
-  top: 0,
-  left: 0,
-  bottom: 0,
-  right: 0,
-  display: 'flex',
-  alignItems: 'center',
-  justifyContent: 'center',
-  overflow: 'auto',
-};
-
-const innerStyle = {
-  margin: 'auto',
-};
+const INNER_ID = 'sb-addon-centered-inner';
+const WRAPPER_ID = 'sb-addon-centered-wrapper';
 
-export default function(storyFn) {
-  const inner = document.createElement('div');
-  Object.assign(inner.style, innerStyle);
+function getOrCreate(id, style) {
+  const elementOnDom = document.getElementById(id);
+
+  if (elementOnDom) {
+    return elementOnDom;
+  }
+
+  const element = document.createElement('div');
+  element.setAttribute('id', id);
+  Object.assign(element.style, style);
+
+  return element;
+}
 
-  const wrapper = document.createElement('div');
-  Object.assign(wrapper.style, style);
+function getInnerDiv() {
+  return getOrCreate(INNER_ID, styles.innerStyle);
+}
+
+function getWrapperDiv() {
+  return getOrCreate(WRAPPER_ID, styles.style);
+}
+
+export default function(storyFn) {
+  const inner = getInnerDiv();
+  const wrapper = getWrapperDiv();
   wrapper.appendChild(inner);
```

Comment:
I think we can assume that if `inner` exists, it's already appendend to `wrapper`